owncloudApp/src/main/java/com/owncloud/android/presentation/files/details/FileDetailsFragment.kt

```diff
@@ -217,6 +217,7 @@ class FileDetailsFragment : FileFragment() {
         }
         startListeningToOngoingTransfers()
         fileDetailsViewModel.checkOnGoingTransfersWhenOpening()
+        activity?.title = getString(R.string.details_label)
```

Comment:
In these cases I'd say is better to use `requireActivity()` so that we avoid handling nullability, since we know this fragment should be always attached to an activity

Reply:
Done